connectivity/mbedtls/platform/src/timing_mbed.cpp

```diff
@@ -1,6 +1,7 @@
 /*
  *  timing.cpp
  *
+ *  Copyright The Mbed TLS Contributors
```

Comment:
Given that the previous commit switched away from using `timeval`, does the commit message for "mbedtls: Add full platform implementation of timing" still make sense? Probably we don't have `error: field has incomplete type 'struct timeval'` anymore.